### DISCUSSION TITLE
Selecting a category clears the message body

### DISCUSSION THREAD

**Discussion by twekberg** [community/community#7862] [Other Feature Feedback, Questions, & Ideas]
Bug

When adding a message to https://github.com/github/feedback/discussions/, if I enter the body and then select a new category, the body gets cleared. Before changing the category one needs to know to save the body somewhere else, and paste it back after selecting a new category. My behavior suggestion is to not clear the body when selecting a new category.

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
